index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/b957cbbe16.js" crossorigin="anonymous"></script>
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="mobile-container">
+    
+    <nav>
+    <div> 
+      <p class="viewmobile">Welcome
+        <i class="fa-solid fa-bars"></i>
+      </p>
+     
+    </div>
+     
+    </nav>
+
+
+  </header>
+  
+    <main class="info">
+      <h2>Hey there. I’m <br> Mandela <br>
+ <span> I’m a software developer </span>
+</h2>
+  <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.
+  </p>
+
+</main>
```

Comment:
Kindly try to match the `background image` with the Figma design and Please add the `arrow button`.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/158698953-3ea0803e-830f-4a0e-9cea-00a61f315fba.png) | ![Template 4-main-mobile - Copy - Copy](https://user-images.githubusercontent.com/24703171/158699325-341fc59e-f366-447e-be72-39d777cb869e.png) |